GhostNetwork.Gateway.Infrastructure/NewsFeedStorage.cs

```diff
@@ -206,6 +196,16 @@ private static PublicationComment ToDomain(Comment entity)
                     : null);
         }
 
+        private static PublicationComment ToDomain(Comment entity)
```

Comment:
Warning

Reply:
It is not fixable)
Static members should appear before non-static members
Our static method is private so if I bring him up I'll get another warn "'public' members should come before 'private' members"

Reply:
It might be discussed but not as part of this request
You might rise another issue